includes/shapes.php

```diff
@@ -25,6 +25,11 @@ class Shapes {
 	 */
 	const FILTER_INCLUDE = 'include';
 
+	/**
+	 * The shape name pattern.
+	 */
+	const SHAPE_NAME_PATTERN = '/^[a-zA-Z-]+$/';
```

Comment:
If the file name contains digits it will be a issue. Did we do it on purpose?

Reply:
@mykytamurzin 
maybe allow 3rd parties to change that?